client/gutenberg/extensions/presets/jetpack/index.json

```diff
@@ -4,11 +4,11 @@
     "map",
     "markdown",
     "publicize",
-    "simple-payments"
+    "simple-payments",
+    "tiled-gallery"
```

Comment:
I moved this here just because beta blocks wasn't working on my setup, will need to look into it closer later.

Reply:
Moved back to beta.